go.mod

```diff
@@ -25,4 +25,5 @@ require (
 	golang.org/x/text v0.3.6 // indirect
 	gopkg.in/ini.v1 v1.63.2 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
Is it possible to remove `v2.4.0` altogether? I thought this version was implicated by dependabot.

Reply:
If only v3 is affected, why is `3.0.0` listed as the patched version? [That was the first release of v3](https://pkg.go.dev/gopkg.in/yaml.v3?tab=versions), and `3.0.1` was not released until after the Dependabot guidance.

Reply:
> I thought this version was implicated by dependabot.

No, the alert points out the issue in `go.sum`. Checking the dependency graph confirms this.

> Is it possible to remove v2.4.0 altogether?
> However, it will be nice to clean up the dependencies if only v3 is used.

Trying to keep PRs small and focused so can upgrade deps in a future PR.

> If only v3 is affected, why is 3.0.0 listed as the patched version? [That was the first release of v3](https://pkg.go.dev/gopkg.in/yaml.v3?tab=versions), and 3.0.1 was not released until after the Dependabot guidance.

It sounds like you're mixing the module name and the release version? This package's _name_ is _v3_ , but as of release `3.0.0` and later the issue should be fixed.. does that help?